tests/ballerina-unit-test/src/test/resources/test-src/expressions/async/basic-async-operations.bal

```diff
@@ -79,8 +79,8 @@ function testAsyncNonNativeBasic9() returns int {
 
 function testAsyncNonNativeBasic10() returns any {
   future<int> f1 = start addNum(5, 2);
-  future<int> f2 = start addNum(5, 2);
-  boolean cancelled_f1 = f1.cancel();
+  future<int> f2 = start subtract(5, 2);
+  boolean cancelled_f2 = f2.cancel();
```

Comment:
do we have a test for wait for all with all futures canceled?

Reply:
No we don't have I'll add one

Reply:
Added a test with f5c0019. Thank you :)